### PR TITLE
Remove unused libraries

### PR DESCRIPTION
**Pull request by sanjaydatasciencedojo**
This will remove unused libraries from the code.

### CODE REVIEW

File: hubconf.py [hunk 1]
```diff
@@ -4,6 +4,4 @@
 # This source code is licensed under the CC-by-NC license found in the
 # LICENSE file in the root directory of this source tree.
 #
-from models import deit_tiny_patch16_224, deit_small_patch16_224, deit_base_patch16_224
```
Comment (zhiqwang) [inline hubconf.py:7]: Hi, I think that these line cannot be removed, it's essential for `torch.hub` to be working.

Comment (fmassa) [reply hubconf.py:7]: As @zhiqwang mentioned, torchhub requires this import to work, so can you keep them?

File: main.py [hunk 2]
```diff
@@ -24,7 +24,6 @@
 from datasets import build_dataset
 from engine import train_one_epoch, evaluate
 from samplers import RASampler
-import models
```
Comment (fmassa) [inline main.py:27]: This import is necessary to register the newly added models to `timm`, otherwise those models will not be available

**Review by fmassa — CHANGES_REQUESTED** [review]
Thanks for the PR!

I've left two comments which require the imports, the others seems fine to remove.

**Comment by sanjaydatasciencedojo** [comment]
The pull request is updated.

**Review by fmassa — APPROVED** [review]
Thanks!

**Pull request merged**